makefiles/master/rules.mk

```diff
@@ -184,6 +217,25 @@ else
 	$(ERROR_BEGIN) "You don't have the GitHub CLI installed. For more information, refer to: https://cli.github.com/" $(ERROR_END)
 endif
 
+# The SPM config is a simple key-value file used to pass build settings to Package.swift.
+# Each line is either empty or starts with a (unique) key, followed by an equals sign, 
+# followed by the key's value.
+spm::
+	@$(PRINT_FORMAT_MAKING) "Creating SPM config (beta)"
```

Comment:
Are we still considering this feature a "beta?"

Reply:
Good point, might be time for this to graduate :)